atlp-rwanda/e-comm-team-techsmith-fn
# #185528211 Redesign of the seller dashboard

## What does this PR do?
This PR aims at redesigning the dashboard of the seller
## Description of Task to be completed?
This task focuses on the implementation of the seller's dashboard design on both mobile and bigger screens(responsiveness)
## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout bg-dashboard-design-185528211`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- Check out the components documentation: `npm run storybook`

## What are the relevant pivotal tracker/Trello stories?
[#185528211](https://www.pivotaltracker.com/story/show/185528211)
## Screenshots (if appropriate)
![Screen Shot 2015-11-07 at 1 07 03 PM](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/108126328/f670c75b-6cd2-4d5b-ba8f-77384b7d095e)
![Screen Shot 2015-11-07 at 1 32 46 PM](https://github.com/atlp-rwanda/e-comm-team-techsmith-fn/assets/108126328/79059de2-00ae-4874-a2de-a1ddac3fb32b)
